### PR TITLE
Colorblind theme

### PR DESCRIPTION
**Pull request by arthurian**
This PR adds the ability to enhance the readability of texts by modifying the existing color/category mappings from their defaults (`green—blue—purple—orange—black`). This change is only applied to the _Visible Vocabulary Tool_ so far, but the implementation should allow it to be extended to all of the other tools. This is intended as a step toward enhancing the overall accessibility of the site for users with color blindness. 

## Changes

- Updated `static/css/base.css` with `theme-default` and `theme-colorblind` classes. 
    - The initial color values for the `colorblind` theme have been provided by Steven Clancy.
    - Some adjustments may be needed once further testing has been completed.
- Added `theme.js` to automatically set the theme on the HTML `body` element and also retrieve the computed level colors based on the active theme, since some JS libraries need the actual color values.
- Updated `templates/theme_base.html` to apply the `default` theme initially and include `theme.js` so all pages can use the `setTheme(name)` function to switch to a different theme.
- Added checkbox option to `text_parsing_analysis.html` so the user can choose to enable the alternate color scheme for word categories/levels.

## Note about Copy & Paste 

It's necessary to apply the CSS color value directly (not referencing a CSS var) in order to preserve copy & paste behavior, particularly when copying & pasting into a **Word document** (a common task with this tool). In other words, the colors will only come over if the color is specified directly (e.g. `color: green` instead of `color: var(--level1-color)`). Hopefully this will be fixed in the future so that we can just use the CSS vars.

```css
.theme-default .level1,
.theme-default [data-level='1'],
.theme-default [data-level='1E'] {
    color: var(--level1-color);
    color: green;  /* redundant, but necessary to support copy & paste into MS Word */
}
```

## Example Screenshots

|Default|Colorblind|
|--|--|
|Screen Shot 2023-03-27 at 11 58 58 AM|Screen Shot 2023-03-27 at 11 58 34 AM|

@eelegiap @Tanvez

### CODE REVIEW

File: parser_tool/static/js/src/lib/theme.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+    function setDefaultTheme() {
+        setTheme('default');
+    }
+
+    function getLevelColors() {
+        // Level colors are defined in CSS vars in base.css.
+        // The current level color depends on the theme applied to the body.
+        const style = getComputedStyle(document.body);
+        return [
```
Comment (Tanvez) [inline parser_tool/static/js/src/lib/theme.js:22]: Cool way to get the colors from CSS into Javascript!

**Review by Tanvez — APPROVED** [review]
Looks good to me!

**Comment by elliottyates** [comment]
@arthurian that's interesting about copy-paste. Is that a limitation of browser support for CSS vars?

**Comment by eelegiap** [comment]
Looked over the changed files and everything looks good to me, thanks @arthurian !

**Pull request merged**